Fixed alignment issue due to existing layout.

zeldafan89 opened the pull request:
Fixed alignment issue due to existing layout being out of date.

zeldafan89 committed:
Fixed alignment issue due to existing layout.

Fixed alignment issue due to existing layout being out of date.

Migushthe2nd committed:
Update details.json

Migushthe2nd reviewed (changes requested):
Please copy the footer from the old image to the new image.

Migushthe2nd committed:
Update details.json

zeldafan89 committed:
Fixed prior issues

Fixed prior issues with the image, game icons and "All Software" text not being aligned properly.

zeldafan89 reviewed (commented):
Submitted the correct overly.png that shows the footer.

zeldafan89 committed:
Fixed prior issues for Right All Apps Rounded

Fixed prior issues with the image, game icons and "All Software" text not being aligned properly.

Migushthe2nd committed:
Update details.json

Migushthe2nd committed:
Update details.json
